src/Compilers/VisualBasic/Portable/Parser/BlockContexts/CompilationUnitContext.vb

```diff
@@ -4,6 +4,8 @@ Imports Microsoft.CodeAnalysis.Syntax.InternalSyntax
 Imports Microsoft.CodeAnalysis.Text
 Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
 Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax.FeatureUtils
```

Comment:
Is this `Imports` necessary, here or in other files where the use is within `Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax`?